src/clojure/puppetlabs/ssl_utils/core.clj

```diff
@@ -574,6 +574,15 @@
   (with-open [r (reader pem)]
     (SSLUtils/pemToCerts r)))
 
+(schema/defn ^:always-validate pem->ca-cert :- X509Certificate
+  "Given a CA certificate bundle and public key, extract the first certificate and
```

Comment:
We expect our certificate to be the first certificate in the bundle. Doesn't that make it a chain? If we expect an unordered bundle here should we have a different implementation of `pemToCaCert`?